src/scripts/components/share/share.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FunctionComponent, useState} from 'react';
+
+import {YoutubeIcon} from '../icons/youtube-icon';
+import {TwitterIcon} from '../icons/twitter-icon';
+import {InstagramIcon} from '../icons/instagram-icon';
+import {FacebookIcon} from '../icons/facebook-icon';
+import {CopyIcon} from '../icons/copy-icon';
+import {ShareIcon} from '../icons/share-icon';
+
+import styles from './share.styl';
+import {FormattedMessage} from 'react-intl';
+import Button from '../button/button';
+import {CloseIcon} from '../icons/close-icon';
+import Overlay from '../overlay/overlay';
+
+const Share: FunctionComponent = () => {
+  const [showShare, setShowShare] = useState(false);
+  const currentUrl = window.location.href;
+  const shareUrl = `ESA Climate From Space: ${window.location.href}`;
+  const urlInput = document.querySelector('#urlInput') as HTMLInputElement;
+  const copyUrl = () => {
+    if (navigator.clipboard) {
+      navigator.clipboard.writeText(shareUrl);
+    } else {
+      urlInput.select();
+      document.execCommand('copy');
+    }
+  };
+
+  return (
+    <div className={styles.share}>
+      <Button
+        className={styles.shareButton}
+        icon={ShareIcon}
+        onClick={() => setShowShare(true)}
+      />
+      {showShare && (
+        <Overlay showCloseButton={false} onClose={() => {}}>
+          <div className={styles.shareOverlay}>
+            <Button
+              icon={CloseIcon}
+              className={styles.closeButton}
+              onClick={() => setShowShare(false)}
+            />
+            <h1 className={styles.title}>Share with the world</h1>
+            <div className={styles.shareButtons}>
+              <a href={'#'} target={'blank'} className={styles.button}>
```

Comment:
Not sure what exactly should be shared on youtube or instagram.

Reply:
Also no idea! Youtube makes no sense and I have no idea how instagram works :)